[CI] CoreFullClusterRestartIT testRecovery failing against upgraded cluster

There have been 5 of these this week in `master`, `7.x` and `7.6`.

```
java.lang.AssertionError: 
mismatch while checking for translog recovery
testrecovery 0 existing_store done 0
testrecovery 0 peer           done 0
testrecovery 1 existing_store done 0
testrecovery 1 peer           done 0
testrecovery 2 existing_store done 0
testrecovery 2 peer           done 0
testrecovery 3 existing_store done 0
testrecovery 3 peer           done 0
testrecovery 4 existing_store done 0
testrecovery 4 peer           done 0
 expected:<true> but was:<false>
at __randomizedtesting.SeedInfo.seed([501B62971D61B9E6:91EB1B3B30317341]:0)
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.failNotEquals(Assert.java:834)
at org.junit.Assert.assertEquals(Assert.java:118)
at org.elasticsearch.upgrades.FullClusterRestartIT.testRecovery(FullClusterRestartIT.java:800)
```

https://gradle-enterprise.elastic.co/s/5y4zpqvjppqxk/tests/futb4ms7jdaog-gweza6zb3vqbg
https://gradle-enterprise.elastic.co/s/tdf4altcwoehc/tests/azvgwapql5okk-gweza6zb3vqbg
https://gradle-enterprise.elastic.co/s/cs3p4ljhknz4a/tests/4taevsg7bdssu-gweza6zb3vqbg
https://gradle-enterprise.elastic.co/s/lqqvda5lljnh2/tests/4taevsg7bdssu-gweza6zb3vqbg
https://gradle-enterprise.elastic.co/s/r2yx7lqnmipfy/tests/azvgwapql5okk-gweza6zb3vqbg

Another one on the `master` pile https://gradle-enterprise.elastic.co/s/5y4zpqvjppqxk
```
REPRODUCE WITH: ./gradlew ':x-pack:qa:full-cluster-restart:v8.0.0#upgradedClusterTest' --tests "org.elasticsearch.xpack.restart.CoreFullClusterRestartIT.testRecovery" -Dtests.seed=ED932745F6998539 -Dtests.security.manager=true -Dtests.locale=vi -Dtests.timezone=Africa/Tripoli -Dcompiler.java=13	
  2> java.lang.AssertionError: mismatch while checking for translog recovery	
    testrecovery 0 existing_store done 0	
    testrecovery 0 peer           done 0	
     expected:<true> but was:<false>	
        at __randomizedtesting.SeedInfo.seed([ED932745F6998539:2C635EE9DBC94F9E]:0)	
        at org.junit.Assert.fail(Assert.java:88)	
        at org.junit.Assert.failNotEquals(Assert.java:834)	
        at org.junit.Assert.assertEquals(Assert.java:118)	
        at org.elasticsearch.upgrades.FullClusterRestartIT.testRecovery(FullClusterRestartIT.java:737)
```

--- a/x-pack/qa/full-cluster-restart/build.gradle
+++ b/x-pack/qa/full-cluster-restart/build.gradle
@@ -70,7 +70,7 @@ for (Version bwcVersion : bwcVersions.indexCompatible) {
 
       setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
       // some tests rely on the translog not being flushed
-      setting 'indices.memory.shard_inactive_time', '20m'
+      setting 'indices.memory.shard_inactive_time', '60m'
       setting 'xpack.security.enabled', 'true'
       setting 'xpack.security.transport.ssl.enabled', 'true'
       setting 'xpack.license.self_generated.type', 'trial'

--- a/x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/CoreFullClusterRestartIT.java
+++ b/x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/CoreFullClusterRestartIT.java
@@ -22,9 +22,4 @@ public class CoreFullClusterRestartIT extends FullClusterRestartIT {
                 .build();
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/52031")
-    @Override
-    public void testRecovery() throws Exception {
-        super.testRecovery();
-    }
 }

